requirements.txt

```diff
@@ -12,7 +12,7 @@ pytz==2022.7.1
 schedule==1.1.0
 simplejson==3.18.1
 smart-open==4.2.0
-SQLAlchemy==2.0.1
+SQLAlchemy==1.4.46
```

Comment:
I am downgrading SQLAlchemy to match other apps of the DevOps portfolio. 😛  E.g., https://github.com/ets-berkeley-edu/damien/blob/dev/requirements.txt#L15